core/data/build.gradle.kts

```diff
@@ -16,6 +16,7 @@ dependencies {
 
     implementation(libs.hilt.android)
     implementation(projects.core.log)
+    implementation(project(mapOf("path" to ":core:database")))
```

Comment:
:core:data module should not implement :core:database dependency, it should use :core:data-source:api